apps/projects/signals.py

```diff
@@ -93,5 +93,16 @@ def increase_poll_answers_count(sender, instance, created, **kwargs):
 
         insight, _ = ProjectInsight.objects.get_or_create(project=project)
         insight.poll_answers += 1
-        insight.active_participants.add(instance.creator.id)
+        if instance.creator:
+            insight.active_participants.add(instance.creator.id)
```

Comment:
I would include only this check for now. The wider question of whether to include unregistered participants in the insights is not part of this story scope. It needs to be discussed with @TeaJayyy in a later stage. So I suggest we leave this PR as a draft. And you open a new PR with a small fix.

Reply:
@m4ra As discussed with @goapunk, we need an answer (or small fix) for that issue now, before we release the new poll. The solution with some inaccuracies but still a somewhat valid representation of users that participated should be fairly doable for now, right? 

I agree - we can implement a more sophisticated solution later.